pages/science/kinetics.rst

```diff
@@ -384,14 +390,28 @@ Surface reactions can be defined in the CTI format using the
 the ``coverage`` keyword argument supplied to the :cti:class:`Arrhenius`
 directive. In the YAML format, surface reactions are identified by the presence
 of surface species and support several
-:ref:`additional options <sec-yaml-interface-reaction>`.
+:ref:`additional options <sec-yaml-interface-Arrhenius>`.
+
+.. TODO: Update the link once version 2.6 is released
 
-Sticking Coefficients
-~~~~~~~~~~~~~~~~~~~~~
+In YAML, the surface reaction ``type`` defaults to ``interface-Arrhenius``, where
+the rate expression uses the
+:ref:`Arrhenius <https://cantera.org/documentation/dev/sphinx/html/yaml/reactions.html#sec-yaml-interface-Arrhenius>`__
+parameterization. As an alternative, Cantera also supports the
+``interface-Blowers-Masel`` surface reaction ``type``, which uses the
+:ref:`Blowers-Masel <https://cantera.org/documentation/dev/sphinx/html/yaml/reactions.html#sec-yaml-interface-Blowers-Masel>`__
+parameterization *(New in Cantera 2.6)*. The prefix ``interface-`` is
+not required as it is inferred automatically.
```

Comment:
I think it's probably better not to specify that the prefix is not required; explicit is better than implicit, and this reads as a bit confusing to me ("where is the `interface-` not required?").

Reply:
I agree that it can be omitted here.